assets/js/cell/live_editor.js

```diff
@@ -250,6 +250,7 @@ class LiveEditor {
 
       return this.__asyncIntellisenseRequest("completion", {
         hint: lineUntilCursor,
+        editor_auto_completion: settings.editor_auto_completion,
```

Comment:
I didn't find any way to detect if the completion was triggered with ctrl+space or requested automatically by the editor. So when auto popups are enabled we don't show the "No runtime" message, even on ctrl+space.